workflows/miassembler.nf

```diff
@@ -97,7 +118,7 @@ workflow MIASSEMBLER {
     } else if ( params.assembler == "megahit" ) {
```

Comment:
It would be ideal if we could incorporate the read type conditional here as well. If single-end, always run MEGAHIT

Reply:
Yes, it's part of the parameters that will be set automatically and just forced with --assembler. I'll keep this open as a reminder
[single/paired end returned by meta]